Assets/Scripts/StateMachines/PlushieDamageSM.cs

```diff
@@ -0,0 +1,72 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlushieDamageSM : BaseStateMachine
+{
+    // State of plushie damage as a small rip
+    [HideInInspector]
```

Comment:
Instead of making these public and hiding them, making these variables up here private, and add a public property for each member variable. I tend to add the public properties at the bottom of the file.